commands/post.go

```diff
@@ -52,7 +52,7 @@ func init() {
 func postCreateCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 	message, _ := cmd.Flags().GetString("message")
 	if message == "" {
-		return errors.New("Message cannot be empty")
+		return errors.New("message cannot be empty")
```

Comment:
This change proposal is being discussed in https://github.com/mattermost/mmctl/pull/62, and although I agree on applying it, I think we should manage it outside of this PR

Reply:
Sure, I will revert these changes 👍